src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react'
+import { Box, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type objectType = {
+  id: string
+  location: string
+  picture: string
+  fName: string
+  lName: string
+}
+const NearMeProfiles = ({ isMobile }: isMobileProps) => {
+  const { classes } = useStyles()
+  const [list, setList] = useState<Array<objectType>>([])
+  const columns = isMobile ? 3 : 6
```

Comment:
amount of columns - create enums.ts file and move values there

Reply:
Values for the number of columns when viewing on mobile and wide screens have been moved to an enums file.